### PR TITLE
API - added business address in search sync

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*https://github.com/bcgov/entity/issues/20026

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: btr-api/src/btr_api/resources/submission.py [hunk 1]
```diff
@@ -127,26 +127,26 @@ def create_register():
         # save before attempting invoice creation so that we can log the id for ops if there's an error
         submission.save_to_session()
         try:
+            # NOTE: this will be moved out of this api once lear filings are linked
```
Comment (hfekete) [inline btr-api/src/btr_api/resources/submission.py:130]: is there a ticket, ticket number would be good if there is :D

File: btr-api/src/btr_api/services/entity.py [hunk 2]
```diff
@@ -55,14 +55,15 @@ def init_app(self, app: Flask):
...
+    def get_entity_info(self, user_jwt: JwtManager, path: str) -> requests.Response:
+        """Get the entity info for the given path."""
         try:
             # make api call
             token = user_jwt.get_token_auth_header()
 
             headers = {'Authorization': 'Bearer ' + token, 'Content-Type': 'application/json'}
-            resp = requests.get(url=self.svc_url + '/businesses/' + identifier,
+            self.app.logger.debug('Getting legal-api data for: %s', path)
+            resp = requests.get(url=self.svc_url + '/businesses/' + path,
```
Comment (hfekete) [inline btr-api/src/btr_api/services/entity.py:66]: What do the path can consist off ?
I understand this path is suffix after that describes business and its sub information, like address or ?

**Review by hfekete — APPROVED** [review]
Looks good, left few questions.

File: btr-api/src/btr_api/resources/submission.py (hunk 1 above)
Comment (kialj876) [reply btr-api/src/btr_api/resources/submission.py:130]: I know I tried to something on the board, but we don't even have an epic for it yet (it'll be in next PI). It's on my list though so it'll get done

File: btr-api/src/btr_api/services/entity.py (hunk 2 above)
Comment (kialj876) [reply btr-api/src/btr_api/services/entity.py:66]: yeah I will add a comment so its clear. It will be identifier + / + sub info suffix (i.e. address)

Comment (hfekete) [reply btr-api/src/btr_api/services/entity.py:66]: As long as its understood that its a url suffix I think we are good, whatever is easiest.

**Pull request merged**